torchtext/vocab/vocab_factory.py

```diff
@@ -34,19 +36,27 @@ def vocab(ordered_dict: Dict, min_freq: int = 1) -> Vocab:
         >>> unk_token = '<unk>'
         >>> default_index = -1
```

Comment:
we could move this close to V3 since it is no longer used in V2

Reply:
Done. Saw that v2 is not needed. So made v3 as v2